types/mui-datatables/index.d.ts

```diff
@@ -111,36 +110,34 @@ export interface MUIDataTableFilterOptions {
 }
 
 export interface MUIDataTableColumnOptions {
-    display?: Display;
+    customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (s: any, c: any, p: any) => any) => string | React.ReactNode;
+    customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string | React.ReactNode;
+    display?: 'true' | 'false' | 'excluded';
+    download?: boolean;
     empty?: boolean;
     filter?: boolean;
     filterList?: string[];
-    filterOptions?: MUIDataTableFilterOptions;
-    filterType?: FilterType;
-    sort?: boolean;
-    searchable?: boolean;
-    sortDirection?: SortDirection;
-    print?: boolean;
-    download?: boolean;
-    viewColumns?: boolean;
+    filterOptions?: string[];
     hint?: string;
-    customFilterListRender?: (value: any) => string;
-    customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string | React.ReactNode;
-    customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (s: any, c: any, p: any) => any) => string | React.ReactNode;
+    print?: boolean;
+    searchable?: boolean;
     setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => object;
+    sort?: boolean;
+    sortDirection?: 'asc' | 'desc';
 }
 
 export interface MUIDataTableOptions {
     caseSensitive?: boolean;
     count?: number;
-    customRowRender?: (data: any[], dataIndex: number, rowIndex: number) => React.ReactNode;
     customFooter?: (
         rowCount: number,
         page: number,
         rowsPerPage: number,
         changeRowsPerPage: () => any,
         changePage: number
     ) => React.ReactNode;
+    customRowRender?: (data: any[], dataIndex: number, rowIndex: number) => React.ReactNode;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface